idl/PeerConnection.idl

```diff
@@ -274,7 +274,7 @@ namespace org
       /// Adds a new track to the RTCPeerConnection, and indicates that it is
       /// contained in the specified MediaStreams.
       /// </summary>
-      RTCRtpSender addTrack(MediaStreamTrack track) throws (RTCError);
+      RTCRtpSender addTrack(MediaStreamTrack track, StringList streams) throws (RTCError);
```

Comment:
>StringList streams [](start = 52, length = 18)

This change will break most apps using m71. How about versioning this API?

Reply:
Blocking this PR breaks the `releases/m71` branch because all related changes have already been submitted to other submodules, so now `windows/wrapper/impl_org_webRtc_MediaStream.cpp` is not found.